contribs/gnodev/main.go

```diff
@@ -71,6 +85,13 @@ func (c *devCfg) RegisterFlags(fs *flag.FlagSet) {
 		"web server listening address",
 	)
 
+	fs.StringVar(
+		&c.nodeRPCListenerAddr,
+		"node-rpc-listener",
+		defaultDevOptions.nodeRPCListenerAddr,
+		"gnoland rpc node listening address",
+	)
+
 	fs.BoolVar(
 		&c.minimal,
 		"minimal",
```

Comment:
minimal? should be no-watch?

Reply:
wdym, it's a different flag?

Reply:
I think it was due to a bad rebase. Resolving it for now.